### PR TITLE
chore(tests): removed deprecation warnings and console

### PR DESCRIPTION
**Pull request by laurentlp**
This PR removes the deprecation warning for `rmdirSync`, bumps jest related dependencies, and removes the console.error from the components tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
   "devDependencies": {
     "@types/jest": "^27.0.3",
     "@types/jest-dev-server": "^5.0.0",
+    "@types/node": "^16.11.13",
```
Comment (samuelmasse) [inline package.json:41]: Why is this here now?

**Review by samuelmasse — APPROVED** [review]
YES. The console.error was really annoying.

File: package.json (hunk 1 above)
Comment (laurentlp) [reply package.json:41]: Because more than one package uses those typings, so it makes more sense that we put it in the root package.json

Comment (samuelmasse) [reply package.json:41]: I'd put them in the engine package instead. Rigth now the server package as mostly all dependencies, but I'll make a cleanup and put the correct deps in the engine and channels package
